board/main.c

```diff
@@ -875,19 +860,8 @@ int main() {
   usb_init();
 
   // default to silent mode to prevent issues with Ford
-#ifdef PANDA_SAFETY
-  can_init(CAN1, 1);
-  can_init(CAN2, 1);
-  #ifdef CAN3
-    can_init(CAN3, 1);
-  #endif
-#else
-  can_init(CAN1, 0);
-  can_init(CAN2, 0);
-  #ifdef CAN3
-    can_init(CAN3, 0);
-  #endif
-#endif
+  for(i=0; i < CAN_MAX; i++)
```

Comment:
When you use `for(int i...)` without setting the c standard to a new enough version, this error happens.

```
main.c: In function 'main':
main.c:875:3: error: 'for' loop initial declarations are only allowed in C99 or C11 mode
   for(int i=0; i < CAN_MAX; i++)
   ^
main.c:875:3: note: use option -std=c99, -std=gnu99, -std=c11 or -std=gnu11 to compile your code
build.mk:71: recipe for target 'obj/main.panda.o' failed
make: *** [obj/main.panda.o] Error 1
```

I did not feel this PR deserved to have a change in the C standard flags so I put the declaration of i at the top of the function as is suggested.